ideahacks/db/data/team.js

```diff
@@ -4,209 +4,215 @@
 
 module.exports = [
 	{
-		departmentName: "Directors",
-		teamPath: "/img/team/directors.png",
-		teamPathRotated: "/img/team/directors-rotated.png",
+		departmentName: "Team Captains",
+		isFourWide: false,
```

Comment:
~~Would it be better to make "isFourWide" a data attribute and attach that to the HTML? My reasoning is that "isFourWide" relates to the styling while this is just data, but if this way makes more sense to you, feel free to keep it. You can put data-is-four-wide="true" in the HTML and then access it in the CSS with [is-four-wide="true"] { /* Selectors here */ }. [Here are the relevant docs](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) if you're interested.~~

Nevermind, I didn't realize you were dynamically rendering these images. Went to the bottom before I realized this wasn't static HTML. I'll keep the paragraph above in case it's useful for something else.